app/controllers/ncr/work_orders_controller.rb

```diff
@@ -29,7 +29,7 @@ def update
 
       super
 
-      if @model_changing && !@model_instance.emergency # skip approvals if emergency
+      unless skip_state_update?
```

Comment:
I guess I am not 100% sure why they matter, which is probably why this is hard for me. Perhaps you can explain? I looked into the controller to see the important for `model_changing` but I am still not clear